packages/augur-ui/src/modules/events/actions/log-handlers.ts

```diff
@@ -170,7 +170,7 @@ export const handleNewBlockLog = (log: Events.NewBlock) => (
   );
   // update assets each block
   if (getState().authStatus.isLogged) {
-    dispatch(updateAssets());
+    dispatch(updateAssets(true));
```

Comment:
this is the only place updateAssets is called the boolean doesn't have any meaning